packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -80,6 +97,136 @@ import 'constants.dart';
 /// }
 /// ```
 /// {@end-tool}
+///
+/// ## Dynamically updating tabs
+///
+/// To dynamically add or remove tabs, TabController has to be updated without
+/// creating a new [AnimationController].
+///
+/// {@tool sample}
+///
+/// A sample application dynamically adds and removes tabs.
+///
+/// ```dart
+/// import 'dart:math' as math;
+///
+/// // import 'package:flutter/material.dart'; // uncomment
```

Comment:
The analyzer somehow treats this as a duplicate import because of the sample right above it. I'm not sure how I can resolve this problem

Reply:
Commenting it out doesn't work either:
```
Error: src/material/tab_controller.dart:114: : Unexpected blank line or "// ..." line near start of subblock in sample code.
```